specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -334,7 +347,7 @@
               }
             },
             "schema": {
-              "$ref": "#/definitions/GroupQuotaSubscriptionId"
+              "$ref": "#/definitions/LROResponse"
```

Comment:
>LROResponse [](http://example.com/codeflow?start=37&length=11)

Why doesn't it return GroupQuotasEntity and provisioningState?

Reply:
I think, it was changed based on the comments in PR: https://github.com/Azure/azure-rest-api-specs-pr/pull/15893
201 responses should have provisioningState.
The response has
"properties": {
"provisioningState": "Accepted"
}

The LROResponse has provisioningState-
 "LROResponse": {
      "type": "object",
      "description": "The provisioning state for the operation.",
      "allOf": [
        {
          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
        }
      ],
      "properties": {
        "properties": {
          "type": "object",
          "properties": {
            "provisioningState": {
              "$ref": "#/definitions/RequestState",
              "readOnly": true
            }
          }
        }
      }
    },